pkg/glob/glob.go

```diff
@@ -142,7 +150,12 @@ func glob(segs []Segment, dir string, cb func(string) bool) bool {
 	for _, info := range infos {
 		name := info.Name()
 		if matchElement(segs, name) {
-			if !cb(dir + name) {
+			name = dir + name
+			info, err := os.Stat(name)
+			if err != nil {
+				return true
+			}
```

Comment:
You already have a `FileInfo`; this is redundant.

Reply:
I have no idea what you're telling me. There is no usable `FileInfo` object at this point in the code.